src/magic_folder/status.py

```diff
@@ -368,3 +372,57 @@ def download_finished(self, folder, relpath):
         """
         del self._folders[folder]["downloads"][relpath]
         self._maybe_update_clients()
+
+
+@attr.s
+class _ProxyDescriptor(object):
+    """
+    Descriptor that returns ``partial(self.<original>.<method>, self.<relative>)``.
+    """
+    original = attr.ib(validator=attr.validators.instance_of(unicode))
+    relative = attr.ib(validator=attr.validators.instance_of(unicode))
+    method = attr.ib(validator=attr.validators.instance_of(string_types))
+
+    def __get__(self, oself, type=None):
+        if oself is None:
+            raise NotImplementedError()
+        original = getattr(oself, self.original)
+        return partial(
+            getattr(original, self.method),
+            getattr(oself, self.relative),
+        )
+
+def relativeProxyFor(iface, original, relative):
```

Comment:
"Our" code is all snake_case right? This probably should be as well.

Reply:
Yeah, I was looking at `proxyForInterface` when I was writing this, and copied the casing there. I've updated it.

Reply:
So this right here is why I find force-pushing to existing PRs hard: I don't see the change here, I have to look at the entire diff again...